alpaca/broker/requests.py

```diff
@@ -405,70 +469,6 @@ def root_validator(cls, values: dict) -> dict:
         return values
 
 
-class UploadW8BenDocumentRequest(NonEmptyRequest):
```

Comment:
Did the black formatter move this?

Reply:
Sorry for the noice. I have moved the location!